tests/Umbraco.AuthorizedServices.Tests/Services/AuthorizedServiceTestsBase.cs

```diff
@@ -8,17 +8,23 @@ internal abstract class AuthorizedServiceTestsBase
 {
     protected const string ServiceAlias = "testService";
 
-    protected Mock<ITokenStorage> TokenStorageMock { get; set; } = null!;
+    protected static Mock<ITokenStorage> TokenStorageMock { get; set; } = null!;
```

Comment:
Not sure if there's a reason for making these properties static?  Fine if there is, but they are generally instance fields when used in a normal test class.  Here they are in a base class, but they could still be instance properties.

Reply:
I think I wanted to include them in `CreateOptionsMonitorServiceDetail` , but gave up on the idea. Corrected.